### ISSUE TITLE
[JavaScript] Issue with regex empty character class

### ISSUE THREAD

**Issue by ghost** [sublimehq/Packages#873]
The following code

```js
console.log(/[]\//.test(""));
```

is highlighted like a comment. Here [] is empty regex character class which is not allowed in Python and PCRE languages, but in JavaScript it is considered as a valid regex pattern. There is currently no use of it, but it still represents correct syntax.

**Comment by wbond** [comment]
Original content:

----

The following code

```js
console.log(/[]\//.test(""));
```

is highlighted like a comment. Here [] is empty regex character class which is not allowed in Python and PCRE languages, but in JavaScript it is considered as a valid regex pattern. There is currently no use of it, but it still represents correct syntax.

**Comment by Thom1729** [comment]
Seems to be fixed by #1009.
